modules/402-ingress-nginx/templates/controller/controller.yaml

```diff
@@ -329,6 +335,15 @@ spec:
                   resource: daemonsets
                   subresource: prometheus-protobuf-metrics
                   name: ingress-nginx
+        {{- if gt $gracePeriod 0 }}
```

Comment:
Do we need the if at all?

Reply:
I agree, it seems it makes sense to add that preStop hook unconditionally